CHANGELOG.md

```diff
@@ -9,6 +9,7 @@ No changes from `alpha-0`.
 ### Added
 - Keyboard controls for the carousel.
 - Pull request template file.
+- `path` parameter in share modals
```

Comment:
Just a small typo, if we stick with a URL, we'll have to update in a future PR. However I'm also thinking we should just pass the slug through to the block and it can construct the URL there.